internal/repositories/vehicle/vehicles.go

```diff
@@ -42,19 +43,27 @@ func (r *Repository) createVehiclesResponse(totalCount int64, vehicles models.Ve
 	nodes := make([]*gmodel.Vehicle, len(vehicles))
 
 	for i, dv := range vehicles {
-		imageURL, err := GetVehicleImageURL(r.Settings.BaseImageURL, dv.ID)
-		if err != nil {
-			wErr := fmt.Errorf("error getting vehicle image url: %w", err)
-			errList = append(errList, gqlerror.Wrap(wErr))
-			continue
+		var imageURI string
+
+		if dv.ImageURI.Valid {
+			imageURI = dv.ImageURI.String
+		} else {
```

Comment:
could we simplify this as:

```suggestion
		imageURI := dv.ImageURI.String
		if !dv.ImageURI.Valid {
```

I think this is done elsewhere as well

Reply:
I'm not sure how this image URI stuff got into the diff here. Unrelated.

Reply:
To be honest I just don't like touching `.String` if it's not valid. I know it's more lines. I'll make the change on main.

Reply:
yeah thats fair, just was thinking from a readability standpoint. I have an aversion to `else`'s

Reply:
Shouldn't appear in the diff anymore.